--- psad-2.0.1/psad.conf~	2006-12-06 04:32:42.000000000 +0200
+++ psad-2.0.1/psad.conf	2006-12-18 01:56:29.105808821 +0200
@@ -443,7 +443,7 @@
 killallCmd       /usr/bin/killall;
 netstatCmd       /bin/netstat;
 unameCmd         /bin/uname;
-whoisCmd         /usr/bin/whois_psad;
+whoisCmd         /usr/bin/whois;
 dfCmd            /bin/df;
 fwcheck_psadCmd  /usr/sbin/fwcheck_psad;
 psadwatchdCmd    /usr/sbin/psadwatchd;
--- psad-2.0.1/install.pl~	2006-12-12 05:01:37.000000000 +0200
+++ psad-2.0.1/install.pl	2006-12-18 02:01:36.612706700 +0200
@@ -348,8 +348,8 @@
         } else {
             die "[*] Could not compile whois";
         }
+		&perms_ownership($WHOIS_PSAD, 0755);
     }
-    &perms_ownership($WHOIS_PSAD, 0755);
     print "\n\n";
 
     ### install perl modules
